Settings/PricePartSettings.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrchardCore.Commerce.Settings
+{
+    public class CurrencySelectionModes
```

Comment:
Why are those strings and not an enum?

Reply:
Good point. Changed now. It's still serialized as strings as it makes it easier to read in the json file.